seed/seed.json

```diff
@@ -666,26 +671,25 @@
                     "BETA",
                     "RELEASE"
                 ],
-                "max_version": "106.1.45.44",
-                "min_version": "104.1.43.54",
+                "min_version": "103.1.43.9",
                 "platform": [
                     "WINDOWS",
                     "MAC",
-                    "LINUX",
-                    "ANDROID"
+                    "LINUX"
                 ]
             },
-            "name": "BraveTranslateStudy"
```

Comment:
I've checked that both translate-related features `BraveTranslateStudy` and `BraveTranslateStudy` are same before/after the PR. The only things is changed here is the order.

BTW, both features are outdated and limited by `max_version`